modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -10,10 +10,13 @@ import scalaz._, Scalaz._
 
 object ObservationDao {
 
-  def insert(o: Observation[StaticConfig, _]): ConnectionIO[Int] =
+  def insert(o: Observation[StaticConfig, Step[DynamicConfig]]): ConnectionIO[Int] =
     for {
       id <- StaticConfigDao.insert(o.staticConfig)
       _  <- Statements.insert(o, id).run
+      _  <- o.steps.zipWithIndex.traverseU { case (s, i) =>
+              StepDao.insert(o.id, Location.unsafeMiddle((i + 1) * 100), s)
```

Comment:
Might want to look at this change.  It seemed surprising to me that insert would add the static config but not the steps, so I added that.